build/phases.go

```diff
@@ -17,6 +17,7 @@ func (l *Lifecycle) Detect(ctx context.Context) error {
 	detect, err := l.NewPhase(
 		"detector",
 		WithArgs(
+			"-log-level", l.determineLogLevel(),
```

Comment:
Based on the results of the acceptance tests we probably want to make this flag optionally set based on lifecycle version to ease the transition (backwards compatibility).

Reply:
oof, good call

Reply:
Fixed in 1f07183496bafccb05050df48a90ad3571d23a4b